### PR TITLE
Add interface to manage events

### PR DESCRIPTION
**Pull request by masolano**
Add new interface to manage events and change visible status of the group box depends of type of user

### CODE REVIEW

File: localMatchApp/sources/homewindow.cpp [hunk 1]
```diff
@@ -21,6 +29,26 @@ HomeWindow::HomeWindow ( QWidget *i_parent ) : QDialog ( i_parent ), ui ( std::m
     /*! It shows buttons in the carousel is called and the index value is sent */
     showButtonsCarousel(indexOfCarousel);
 
+    /*! If the user type is personal the area of manage event is not visible */
+    if ( typeUser == 1 ) {
```
Comment (wfallascr) [inline localMatchApp/sources/homewindow.cpp]: Please, contemplate the option of using an enum class for this.

File: localMatchApp/headers/homewindow.h [hunk 2]
```diff
@@ -47,6 +47,11 @@ class HomeWindow : public QDialog
      */
     void onButtonSubscribeClicked ( );
 
+    /*!
+     * The function when the user click the button of new image is declare
```
Comment (luvalverde) [inline localMatchApp/headers/homewindow.h]: ... is declared

**Review by luvalverde — CHANGES_REQUESTED** [review]

**Comment by masolano** [comment]
PR back to you

**Review by luvalverde — APPROVED** [review]

**Review by wfallascr — CHANGES_REQUESTED** [review]

**Comment by masolano** [comment]
PR back to you

**Review by wfallascr — APPROVED** [review]

**Pull request merged**